packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -257,20 +284,38 @@ export default class PrepaidCard {
       );
     }
 
+    let txnHash = gnosisResult.ethereumTx.txHash;
     if (typeof onTxnHash === 'function') {
-      await onTxnHash(gnosisResult.ethereumTx.txHash);
+      await onTxnHash(txnHash);
     }
 
-    return await waitUntilTransactionMined(this.layer2Web3, gnosisResult.ethereumTx.txHash);
+    return await waitUntilTransactionMined(this.layer2Web3, txnHash);
   }
 
+  // TODO: add overload for passing txnHash
+  async split(txnHash: string): Promise<{ prepaidCards: PrepaidCardSafe[]; txReceipt: TransactionReceipt }>;
   async split(
     prepaidCardAddress: string,
     faceValues: number[],
     customizationDID: string | undefined,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<{ prepaidCards: PrepaidCardSafe[]; txReceipt: TransactionReceipt }>;
+  async split(
+    prepaidCardAddressOrTxnHash: string,
+    faceValues?: number[],
+    customizationDID?: string | undefined,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<{ prepaidCards: PrepaidCardSafe[]; txReceipt: TransactionReceipt }> {
+    if (!faceValues) {
```

Comment:
maybe a better test is that `safeAddressOrTxnHash` actually looks like an address, that way you are asserting on the value you care about rather than making an assertion on a side effect. All addresses have 40 characters plus "0x" (so 42 characters in total). All txn hashes have 64 characters plus "0x" (so 66 chars in total).